iroha-cli/interactive/impl/interactive_transaction_cli.cpp

```diff
@@ -226,8 +226,9 @@ namespace iroha_cli {
       auto res = handleParse<std::shared_ptr<iroha::model::Command>>(
           this, line, command_handlers_, command_params_descriptions_);
 
-      if (not res) {
+      if (not res and not *res) {
         // Continue parsing
+        std::cout << "Enter the value again" << std::endl ;
```

Comment:
Imo it's better write smth like "Cannot parse the result" as long as after that there should be additional message that asks for a prompt